server/sonar-web/src/main/js/app/utils/getCurrentUserFromStore.js

```diff
@@ -17,16 +17,14 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-import { translate } from '../../../helpers/l10n';
+// @flow
+import { getCurrentUser } from '../store/rootReducer';
 
-export default {
-  getLocalizedDashboardName(baseName) {
-    const l10nKey = `dashboard.${baseName}.name`;
-    const l10nLabel = translate(l10nKey);
-    if (l10nLabel !== l10nKey) {
-      return l10nLabel;
-    } else {
-      return baseName;
-    }
-  }
+// TODO remove my usages
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AS1135)